templates/ModelImportSnippet.stencil

```diff
@@ -0,0 +1 @@
+import Foundation
```

Comment:
What is the purpose of this snippet?

Reply:
The Model files does not need AzureCore as dependency. So I create a new snippet for model files only.

Reply:
Hmm... okay. Maybe for the future, we can just pass a list of packages to our render command and they can be rendered with a for loop. Swift imports never get very complex. This is okay for now.